backend/app/tests/services/collections/test_create_collection.py

```
@@ -188,3 +416,226 @@ def test_execute_job_success_flow_updates_job_and_creates_collection(
     docs = DocumentCollectionCrud(db).read(created_collection, skip=0, limit=10)
     assert len(docs) == 1
     assert docs[0].fname == document.fname
+
+
+@pytest.mark.usefixtures("aws_credentials")
+@mock_aws
+@patch("app.services.collections.create_collection.get_openai_client")
+def test_execute_job_assistant_create_failure_marks_failed_and_deletes_vector(
```

Ruff (0.14.3)

Redefinition of unused `test_execute_job_assistant_create_failure_marks_failed_and_deletes_vector` from line 198

(F811)